bgpd/bgp_zebra.c

```diff
@@ -2042,6 +2046,41 @@ static void bgp_zebra_process_local_ip_prefix(int cmd, struct zclient *zclient,
 	}
 }
 
+static void bgp_zebra_process_label_chunk(
+	int cmd,
+	struct zclient *zclient,
+	zebra_size_t length,
+	vrf_id_t vrf_id)
+{
+	struct stream *s = NULL;
+	uint8_t response_keep;
+	uint32_t first;
+	uint32_t last;
+
+	s = zclient->ibuf;
+	STREAM_GETC(s, response_keep);
+	STREAM_GETL(s, first);
+	STREAM_GETL(s, last);
+
+	if (first > last ||
```

Comment:
is not it already handled by Zebra Label Manager ?

Reply:
I think it's a good idea for programs to check their input so that any errors in other programs do not propagate.

Reply:
it is ok to me that behaviour